### ISSUE TITLE
Issues with ./bin directory

### ISSUE THREAD

**Issue by phindmarsh** [kubitron/redmine_git_hosting#22]
I haven't put much testing into this issue, so it's likely its just a configuration problem.

When pulling a new copy of the plugin it appears the ./bin directory is removed from the plugin root. Then when opening the plugin page from within redmine a 500 server error is thrown about the directory missing. 

In the past I could simply `mkdir ./bin` and `chown www-data ./bin` and that would resolve the problem.

I have redmine installed via apt-get on Ubuntu 10.04. The Redmine root (and subdirectories) are (mostly) chmod'd `760` and owned by `root`. Im aware this is probably where the problem lies, and probably not ideal having root own everything. Redmine is served up via Apache using mod_passenger running as `www-data`.

**Comment by kubitron** [comment]
Ok.  You might try pulling down the latest commit.  

Look at steps 7 and 8 (particularly 8) of the step-by-step instructions.  I decided to explicitly address the bin directory.  Does this help?

**Comment by phindmarsh** [comment]
Yep, that seems to make the directory (and fail nicely with a Passenger error if the directory is not there/writable). The only thing I had to do (which I think is related to my redmine instance being installed via apt-get as root) is change ownership to www-data.

However, as a disclaimer, we just moved our Redmine and repositories to a new server, so these steps were run on a fresh install before I migrated the database etc which may affect what the script would do under normal circumstances.

I suppose if you wanted to, add an argument to the `redmine_git_hosting:install_scripts` script to specify the web server user to chown to. Once again, thanks for all your hard work on this, it is much appreciated!
